_api-reference/index-apis/blocks.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+title: Blocks
+parent: Index APIs
+nav_order: 6
+---
+
+# Blocks
+**Introduced 1.0**
+{: .label .label-purple }
+
+The Blocks API limits which operations are available on the specified index. OpenSearch provides different types of blocks that allow you to restrict write, read, or metadata operations on an index. When adding a write block through the API, it ensures that all shards of the index have properly accounted for the block before returning a successful response to the user. This means that any in-flight write operations to the index have been completed before the write block takes effect.
```

Comment:
```suggestion
Use the Blocks API to limit certain operations on a specified index. The different types of blocks allow you to restrict write, read, or metadata operations on an index. 
For example, adding a `write` block through the API ensures that all shards of the index have properly accounted for the block before returning a successful response to the user. Any in-flight write operations to the index must have been completed before the `write` block takes effect.
```

Reply:
I don't really understand why you are calling out the 'write' block here without describing the others. Can you accomplish the same thing by describing it in the table row?

Reply:
We do describe all of the options in the table row on line 25.

Reply:
Yes, since they are in the table, then why call it out in the opening paragraph? If you describe one, you should describe them all. That's why I added the 'for example', because it seemed a bit out of place. But if the 'for example' works, then it should be fine to leave.

Reply:
For example is fine with me.